jupyterhub/handlers/base.py

```diff
@@ -1005,7 +1007,17 @@ def abort():
         self.log.warning(
             "User %s server stopped, with exit code: %s", user.name, status
         )
-        await self.proxy.delete_user(user, server_name)
+        proxy_deletion_start_time = time.perf_counter()
+        try:
+            await self.proxy.delete_user(user, server_name)
+            PROXY_DELETE_DURATION_SECONDS.labels(
+                status=ProxyDeleteStatus.success
+            ).observe(time.perf_counter() - proxy_deletion_start_time)
+        except:
```

Comment:
It's generally a good idea to catch the `Exception` class, not a blank `except`

```suggestion
        except Exception:
```